pkg/hope/remote_kubeadm.go

```diff
@@ -70,7 +71,17 @@ func KubeadmResetRemote(log *logrus.Entry, kubectl *kubeutil.Kubectl, node *Node
 }
 
 func KubeadmGetClusterCertificateKey(log *logrus.Entry, node *Node) (string, error) {
-	output, err := ssh.GetSSH(node.ConnectionString(), "sudo", "kubeadm", "init", "phase", "upload-certs", "--upload-certs")
+	certsCommand := []string{
+		node.ConnectionString(),
+		"sudo",
+		"KUBECONFIG=/etc/kubernetes/admin.conf",
```

Comment:
Removes the need to/assumption that `root` has a `/root/.kube/config` file. Saves the need for nodes to copy it.